src/clientSideScene/CameraControls.ts

```diff
@@ -1321,6 +1346,168 @@ export class CameraControls {
+     * The closer to this value, the more we reduce the calculated zoom transformation to reduce jitter.
+     */
+    const velocityLimit = 0.5
+    /** Amount of pixel delta of calculated zoom transform needed before we send to the engine */
+    const normalizedScaleThreshold = 5
+    const velocityFlickDecay = 0.03
+    /** Refresh rate for flick orbit decay timer */
+    const decayRefreshRate = 16
+    const decayIntervals: ReturnType<typeof setInterval>[] = []
+    const clearIntervals = () => {
+      for (let i = decayIntervals.length - 1; i >= 0; i--) {
+        clearInterval(decayIntervals[i])
+        decayIntervals.pop()
+      }
+    }
+
+    const hammertime = new Hammer(domElement, {
+      recognizers: [
+        [Hammer.Pan, { pointers: 1, direction: Hammer.DIRECTION_ALL }],
+        [
+          Hammer.Pan,
+          {
+            event: 'doublepan',
+            pointers: 2,
+            direction: Hammer.DIRECTION_ALL,
+            threshold: panDistanceThreshold,
+          },
+        ],
+        [Hammer.Pinch, { enable: true, threshold: zoomScaleThreshold }],
+      ],
+    })
+
+    // TODO: get the engine to coalesce simultaneous zoom/pan/orbit events,
+    // then we won't have to worry about jitter at all.
+    hammertime.get('pinch').recognizeWith(hammertime.get('doublepan'))
+
+    // Clear decay intervals on any interaction start
+    hammertime.on('panstart doublepanstart pinchstart', () => {
+      clearIntervals()
+    })
+
+    // Orbit gesture is a 1-finger "pan"
+    hammertime.on('pan', (ev) => {
+      if (this.syncDirection === 'engineToClient' && ev.maxPointers === 1) {
+        if (this.enableRotate) {
+          const orbitMode =
+            this.getSettings?.().modeling.cameraOrbit.current !== 'spherical'
+              ? 'rotatetrackball'
+              : 'rotate'
+          this.moveSender.send(() => {
+            this.doMove(orbitMode, [ev.center.x, ev.center.y])
+          })
+        }
+      }
+    })
+    // Fake flicking by sending decaying orbit events in the last direction of orbit
+    hammertime.on('panend', (ev) => {
+      /** HammerJS's `event.velocity` gives you `Math.max(ev.velocityX, ev.velocityY`)`, not the actual velocity. */
+      let velocity = Math.sqrt(ev.velocityY ** 2 + ev.velocityX ** 2)
+      const center = ev.center
+      const direction = ev.angle
+
+      if (
+        this.syncDirection === 'engineToClient' &&
+        ev.maxPointers === 1 &&
+        this.enableRotate &&
+        velocity > 0
+      ) {
+        const orbitMode =
+          this.getSettings?.().modeling.cameraOrbit.current !== 'spherical'
+            ? 'rotatetrackball'
+            : 'rotate'
+
+        const decayInterval = setInterval(() => {
+          const decayedVelocity = velocity - velocityFlickDecay
+          if (decayedVelocity <= 0) {
+            this.onMouseUpInner(ev.srcEvent as PointerEvent)
```

Comment:
Can this be avoided?
Would 
```suggestion
if (ev.srcEvent instanceof PointerEvent) {
  this.onMouseUpInner(ev.srcEvent)
}
```
work?